test/utils/Transmuter.sol

```diff
@@ -53,11 +53,6 @@ abstract contract Transmuter is Helper {
         facetNames.push("Swapper");
         facetAddressList.push(address(new Swapper()));
 
```

Comment:
why did you remove it?

Reply:
because it's maybe what makes tests so long -> since we have to re-add the facets all the time